package.json

```diff
@@ -21,21 +21,23 @@
     "url": "https://github.com/meetup/meetup-web-mocks/issues"
   },
   "homepage": "https://github.com/meetup/meetup-web-mocks#readme",
-  "dependencies": {
+  "peerDependencies":{
+    "react": "^15.3.2",
```

Comment:
In that case, we need to update the major versions! The errors are 'real' in this case, and the version notation is correctly bringing up a version incompatibility that we weren't aware of until now.